magento/inventory
# Issue #3291: fix console fail on purhuse online payment order place

### Description (*)

Environment:
- Magento EE 2.4.2
- B2B 1.3.1

Added Condition to avid console error in console if shipping step exists.
### Fixed Issues (if relevant)

1. Fixes magento/inventory#3291

### Manual testing scenarios (*)

1. Enable B2B;
2. Enable Purchase Order (in the config and for the company);
3. Enable and configured "In-store Delivery" Ship Method;
4. Make a purchase order with an online payment method. Purchase order after place will be in "APPROVED - PENDING PAYMENT" status.
5. Place this order -> will be redirected to the checkout payment step, to set Credit Card data.
6. Check console.

### Contribution checklist (*)
